changelog.mdx

```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
 
 ---
 
+## October 14, 2024
+
+### SQL explorer in dashboard for datasets
+
+You can now run SQL queries in the dashboard to explore data in [datasets](https://docs.chalk.ai/docs/datasets).
+To use this feature, navigate to the `Datasets` page in the menu, select a dataset,
+and click on the `Output Explorer` tab.
+
+![SQL Explorere for Datasets](/img/sql_explorer_datasets.png)
+
+### Optionally evict nulls from your DynamoDB online store
+
+Last week we enabled the option to decide whether to persist null values for features in Redis lightning
+online stores, and this week we have enabled this feature in DynamoDB online stores. By default, null
+values are persisted in the online store for features defined as `Optional`, but you can set `cache_nulls=False`
+in the `feature` method to evict null values. Read more about how to use the `cache_nulls` parameter
+[here](/api-docs#feature.cache_nulls).
+
+### Set environment variables and more in Advanced section of Cloud Resource Configurations page in dashboard
```

Comment:
```suggestion
### Set environment variables and more in the Advanced section of the Cloud Resource Configurations page on the dashboard
```

Reply:
Gonna try to stay consistent with "in the dashboard"